### DISCUSSION TITLE
How to check if REX prefix is present?

### DISCUSSION THREAD

**Discussion by ItsEthra** [icedland/iced#446] [Q&A] [answered]
If I have an Instruction, is it possible to check if it uses rex prefix? I looked through the docs and couldn't find anything. Event `Instruction::encoding()` returns `Legacy` for rex prefixed instructions. I'm using rust library if that matters.

**Comment by wtfsck** [comment, marked as answer]
There's no function that can do that. Why do you need to know if it has a `REX` prefix?

The only workaround is to parse the bytes yourself.
